scripts/jenkins/ardana/gerrit/build_test_package.py

```diff
@@ -158,6 +169,50 @@ def _create_test_project(self):
                    self.obs_test_project_name, '--accept-in-hours', 720,
                    '-m', 'Auto delete after 30 days.')
 
+    def get_obsinfo_filename(self, package, service_def=None):
```

Comment:
The pattern of code in this method is repeated below in the is_current method, when retrieving the .obsinfo file...

Why not define a helper method. e.g, _cat_ibs_file() which handles the osc cat operation, and then call it in both places? Or even define methods wrapping that helper method, which retrieve the _service and obsinfo file contents

Reply:
done